PushNotifications/PushNotifications.xcodeproj/xcshareddata/xcschemes/PushNotifications.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0930"
+   LastUpgradeVersion = "1130"
```

Comment:
we've got a lot of these lastUpgradeChecks and lastUpgradeVersion - what do those numbers actually mean and how did you increase them?

Reply:
Each new version of Xcode validates existing project settings. Xcode will rise a warning about  the missing validation. After the validation is being performed and accepted by a developer, Xcode updates the above numbers.